pkg/output/template.go

```diff
@@ -0,0 +1,111 @@
+package output
```

Comment:
Renaming `template.go` to `cli_template.go`, or moving all files related to the cli output inside an `output/cli/` directory.
Indeed, this is a really picky comment :D